### PR TITLE
Add.token.split

### PR DESCRIPTION
**Pull request by herman-rogers**

### CODE REVIEW

File: contracts/NewfiAdvisor.sol [hunk 1]
```diff
@@ -163,22 +155,22 @@ contract NewfiAdvisor is
...
 
-    // IMP NOTE -> Right now these function will only give the pool value for the assets locked in mstable and yearn we can then get the total value in graph if needed we can tweak these functions to get the pool value of locked and unlocked assets direclty
     function getStablePoolValue(address _advisor)
         public
         view
         returns (uint256)
     {
         Advisor storage advisor = advisorInfo[_advisor];
         address stablePool = advisor.stablePool;
-        uint256 mstablePoolInvestmentValue = SavingsContract(savingContract)
```
Comment (viraj124) [inline contracts/NewfiAdvisor.sol:148]: why is this function commented, it is needed to get the pool value

File: contracts/NewfiAdvisor.sol [hunk 2]
```diff
@@ -162,15 +162,14 @@ contract NewfiAdvisor is ReentrancyGuardUpgradeSafe, ProxyFactory, Helper {
...
-        //        uint256 mstablePoolInvestmentValue = SavingsContract(savingContract)
-        //            .creditBalances(stablePoolProxy)
-        //            .mul(SavingsContract(savingContract).exchangeRate());
-        //        // whitelisted vault addresses for now
-        //        uint256 yearnPoolInvestmentValue = YearnVault(getUSDCVault())
-        //            .balanceOf(advisor.volatilePool)
-        //            .mul(YearnVault(getUSDCVault()).getPricePerFullShare());
-        //        return mstablePoolInvestmentValue.add(yearnPoolInvestmentValue);
-        return 1;
+        uint256 mstablePoolInvestmentValue = SavingsContract(savingContract)
```
Comment (viraj124) [inline contracts/NewfiAdvisor.sol:165]: okay fine

File: contracts/NewfiAdvisor.sol [hunk 3]
```diff
@@ -71,6 +72,8 @@ contract NewfiAdvisor is
 
     address[] public advisors;
 
+    address[] public advisorTokens;
```
Comment (viraj124) [inline contracts/NewfiAdvisor.sol:69]: do we need this ? we have the struct where we track the advisors both pool tokens

Comment (herman-rogers) [reply contracts/NewfiAdvisor.sol:69]: It's a list of all the tokens in existence, might not be super helpful though. Dunno

File: contracts/NewfiAdvisor.sol (hunk 2 above)
Comment (herman-rogers) [reply contracts/NewfiAdvisor.sol:165]: I think this was just an accident, pretty sure I pushed it up uncommented?

**Review by viraj124 — APPROVED** [review]
LGTM

**Pull request merged**